src/figuresClasses.ts

```diff
@@ -1,19 +1,89 @@
-export interface Figure {
+enum Color {
```

Comment:
this can be just a type, not an enum, because that's how you use it

Reply:
done